### PR TITLE
Update scheduler to work with copy

### PR DESCRIPTION
**Pull request by lucyge2022**
### What changes are proposed in this pull request?
make scheduler universally work for copyjob + doraloadjob

### Why are the changes needed?
resolve conflict of copyjob to use the new scheduler

### Does this PR introduce any user facing changes?
N/A

### CODE REVIEW

**Review by jja725 — APPROVED** [review]
LGTM, leave some minor questions.

File: common/transport/src/main/proto/grpc/block_worker.proto [hunk 1]
```diff
@@ -195,10 +195,11 @@ message LoadFileRequest {
 }
 
 message File{
-  optional string ufs_path = 1;
+  optional string alluxio_path = 1;
```
[the remark this answers is not in the document]
Comment (lucyge2022) [reply common/transport/src/main/proto/grpc/block_worker.proto:198]: actually this is for the listStatus using FileSystem, so i need the alluxiopath back to add to retry files so retry files could use the alluxio path and do fs.getStatus to get the length and ufspath. let me try replace the current iterator with ufs iterator that u have.

File: dora/core/server/master/src/main/java/alluxio/master/job/CopyJob.java [hunk 2]
```diff
@@ -142,6 +147,21 @@ public CopyJob(String src, String dst, boolean overwrite, Optional<String> user,
     mCheckContent = checkContent;
   }
 
+  private static class RoundRobinWorkerAssignPolicy extends WorkerAssignPolicy {
```
[the remark this answers is not in the document]
Comment (lucyge2022) [reply dora/core/server/master/src/main/java/alluxio/master/job/CopyJob.java:150]: done

File: conf/alluxio-site.properties.dora [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
```
[the remark this answers is not in the document]
Comment (lucyge2022) [reply conf/alluxio-site.properties.dora]: already removed

**Comment by jja725** [comment]
alluxio-bot, merge this please

**Pull request merged**